pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -220,28 +247,21 @@ func (c *DeploymentConfigController) findDetails(config *deployapi.DeploymentCon
 		// Check if the image stream tag pointed by the trigger exists
 		if _, err := c.osClient.ImageStreamTags(config.Namespace).Get(name, tag); err != nil {
 			if !errors.IsNotFound(err) {
-				glog.V(2).Infof("Error while trying to get image stream tag %q: %v", istag, err)
-				return details, existingDeployments, latestDeploymentExists, nil
+				return "", fmt.Errorf("couldn't get image stream tag %q: %v", istag, err)
 			}
 			// In case the image stream tag was not found, then it either doesn't exist or doesn't exist yet
 			// (a build configuration output points to it so it's going to be populated at some point in the
 			// future)
-			details = fmt.Sprintf("The image trigger for image stream tag %q will have no effect because image stream tag %q does not exist.", istag, istag)
-			bcList, err := c.osClient.BuildConfigs(kapi.NamespaceAll).List(labels.Everything(), fields.Everything())
-			if err != nil {
-				glog.V(2).Infof("Error while trying to list build configs: %v", err)
-				return details, existingDeployments, latestDeploymentExists, nil
-			}
-
-			for _, bc := range bcList.Items {
+			invalidIsTags = append(invalidIsTags, istag)
```

Comment:
This is still expensive in memory and CPU.  I'd rather just not do this if it's not improving something 50% of users will hit.  How many people will see this error?